accord-core/src/main/java/accord/coordinate/tracking/AbstractTracker.java

```diff
@@ -156,6 +156,7 @@ public boolean any(Predicate<ST> test)
     {
         for (ST tracker : trackers)
         {
+            if (tracker == null) continue;
```

Comment:
tracker relies on a `maxShardsPerEpoch` to find offsets, so for topologies with different amount of shards, this would NPE...

This is kinda from my patch, as you only hit this if you add/remove keyspaces...